apimodels/agent_models.go

```diff
@@ -219,6 +229,9 @@ func (ch *CreateHost) validateAgentOptions() error {
 func (ch *CreateHost) setNumHosts() error {
 	if ch.NumHosts == "" {
 		ch.NumHosts = "1"
+	}
+	if ch.CloudProvider == ProviderDocker && ch.NumHosts != "1" {
+		return errors.New(fmt.Sprintf("num_hosts cannot be greater than 1 for cloud provider %s", ProviderDocker))
```

Comment:
why can't you create more than one? 

also errors.Errorf is more straightforwared than errors.New(fmt.Sprintf())